exchange/utils.go

```diff
@@ -65,7 +66,7 @@ func cleanOpenRTBRequests(ctx context.Context,
 			privacyEnforcement.GDPR = false
 		}
 
-		privacyEnforcement.Apply(bidReq)
+		privacyEnforcement.Apply(bidReq, isAMP)
```

Comment:
Thoughts around setting `isAMP` in the request context and passing the context to the `privacyEnforcement` function? 

Reason being, if we have other specific conditions based on which we want to tweak the privacy enforcement in future, this function signature would keep getting long, making it difficult to understand the purpose of the function from the signature itself and it might be helpful if we send all the information as part of the request context or a privacy specific struct.

Reply:
I think this just a revert of the AMP commit. This looks like how it was written before. I didn't feel that isAMP was a good fit for the `Enforcement` struct and since it was going away soon, I didn't worry. That's obviously not longer the case (at least not for a little bit) - but I'm fine leaving it as is for now. There are no other conditions which I foresee being added. This one is a bit tricky because it's information from the endpoint, and not from the literal OpenRTB request itself.

Reply:
Yeah, I agree that this information coming from the endpoint rather than the OpenRTB request itself makes it a little tricky but IMO, that is the kind of request level information that context can be used to hold and passed along in the code.

Nevertheless, I didn't know that's how this code was before (I have been away too long from Prebid Server 😅) and is part of the revert that this PR is doing so makes sense to keep it as is for this PR 👍